.beatconfig

```diff
@@ -0,0 +1,5 @@
+packetbeat-/packetbeat-
+filebeat-/filebeat-
+topbeat-/topbeat-
+winlogonbeat-/winlogonbeat-
+logstash-/logstash-
```

Comment:
In previous version we had funny values as an example. Original configfile should have sane values. 
Question is where to document how to use it.